module-4/backend/week_one.md

```diff
@@ -5,15 +5,35 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+  Object Oriented JS
+  
 2. What are some tools to help debug JavaScript code?
+ PryJS, Debugger, Console, Node, node debug
+ 
 3. What are some tools you need in order to unit test your JavaScript?
+ Chai and Mocha
+ 
 4. What is the syntax for invoking a function?
+  Function name
+
 5. What's `this` in JavaScript?
+  The abstract objects 'self'/attributes of object
+  
 6. What is Webpack and why is it useful?
+  Pre compiles and mini-fys our JS files into one file
+  
 7. When/why do you want to use event delegation?
+  Event delegation is useful for client tracking, UX/UI, If you don't want to refresh a page, if you want to load a page with certain portions of the page loading before the others, etc.
```

Comment:
you've listed reasons to leverage events and manipulate the DOM. what specifically do we use event delegation for?